app/exporters/odt/cv.rb

```diff
@@ -4,12 +4,15 @@
 module Odt
   class Cv
 
-    def initialize(person)
+    def initialize(person, anon)
       @person = person
+      @anon = (anon != 'false')
```

Comment:
Then change this to `@params = params`.
You can now define a new method like this:
```
def anon
@params[:anon].present?
end
```
and use that method instead of `@anon`